### ISSUE TITLE
Add option to disable multiple arguments with the form `-m a b`

### ISSUE THREAD

**Issue by joeferner** [75lb/command-line-args#52]
If you have default options combined with multiple you cannot put the default options after the multiple argument.

Example
```
{name: 'group', alias: 'g', multiple: true},
{name: 'args', defaultOption: true, multiple: true}
```

When trying to parse `-g group1 -g group2 a b`, the results end up as `group: ['group1', 'group2', 'a', 'b']`. I would like to have it result in `group: ['group1', 'group2'], args: ['a', 'b']`

It would be nice to have an option to disable this type of multiple behavior

**Comment by 75lb** [comment]
> nice to have an option to disable this type of multiple behavior

could you give an example of what effects this option would have please? 

A `multiple` (e.g. `--files *.js` will consume all values up until the next option (e.g. `--verbose`) or end of input. This is the necessary behaviour. You are right, if you also have a `multiple` `defaultOption` defined it does introduce a risk of ambiguity. The solution to this situation is normally in how you define your options but I'm interested to hear more about the solution you have in mind.

**Comment by joeferner** [comment]
```
{name: 'group', alias: 'g', multiple: true, multiplePerSwitch: false},
{name: 'args', defaultOption: true, multiple: true}
```

By providing something like `multiplePerSwitch: false` it would disable `-g group1 group2` behavior and would require you to use `-g group1 -g group2`

**Comment by 75lb** [comment]
ok, sounds fair enough thanks.. I will accept a PR for this or look into it for the next release.

**Comment by 75lb** [comment]
while we wait for this feature, I should add the workaround in your case would be to remove ambiguity by explicitly specifying where the `--args` begin: 

```
$ example -g group1 -g group2 --args a b
```

**Comment by joeferner** [comment]
Thanks, I'll try and get a PR submitted soon

**Comment by 75lb** [comment]
does [this explanation of lazy multiples](https://github.com/75lb/command-line-args/wiki/Lazy-multiples) make sense?

You can play with the feature in the prerelease.

```
$ npm install command-line-args@next
```

**Comment by joeferner** [comment]
Sounds good to me.

**Comment by 75lb** [comment]
Implemented in [v5.0.0](https://github.com/75lb/command-line-args/releases/tag/v5.0.0).
